Signal/src/AppDelegate.m

```diff
@@ -274,23 +275,25 @@ - (BOOL)application:(UIApplication *)application
         NSString *filename = url.lastPathComponent;
         if ([filename stringByDeletingPathExtension].length < 1) {
             DDLogError(@"Application opened with URL invalid filename: %@", url);
-            [self showErrorAlertWithTitle:
```

Comment:
I've added a utility function for showing simple alerts.  

* This will DRY up boilerplate.
* We're often not presenting alerts in a safe way, so some alerts are not appearing.  

I've made a card to apply these utility methods in more places.